snap/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: branch-diff
+version: git
+summary: Compares 2 git branches without hassle.
+description: |
+    A tool to list print the commits on one git branch
+    that are not on another using loose comparison.
+grade: stable
+confinement: strict
+
+apps:
+  branch-diff:
+    command: bin/branch-diff
+    plugs: [network, home]
+
+parts:
+  branch-diff:
+    source: .
+    plugin: nodejs
+    stage-packages: [git]
+    build-packages: [zlib1g-dev]
```

Comment:
why does this need zlib1g-dev during build?

Reply:
Without it we were getting an error on build.snapcraft.io. But the logs are too old now and I can't access them anymore. I tried a clean build using a container locally, and it succeeded without the build package, so I have removed it. We can always add it back if necessary, and if there are different results locally and on build.snapcraft.io it means we have a bug that I'll need to investigate :) I'll keep an eye here.